### PR TITLE
feat: document image puller default image workflow

### PR DESCRIPTION
**Pull request by dkwon17**
Signed-off-by: David Kwon <dakwon@redhat.com>

> Read our [Contribution guide](https://github.com/eclipse/che-docs/blob/master/CONTRIBUTING.adoc) before submitting a PR.

### What does this PR do?
Adds documentation to http://localhost:4000/che-7/administration-guide/installing-image-puller-using-che-operator about the image puller default images support.

### What issues does this PR fix or reference?
The documentation is about the fixes to these issues:
https://github.com/eclipse/che/issues/20081
https://github.com/eclipse/che/issues/19657

### Specify the version of the product this PR applies to.
7.33

### PR Checklist

As the author of this Pull Request I made sure that:

- [x] `vale` has been run successfully against the PR branch
- [x] Link checker has been run successfully against the PR branch
- [ ] Documentation describes a scenario that is already covered by QE tests, otherwise an issue has been created and acknowledged by Che QE team
- [ ] Changed article references are updated where they are used (or a redirect has been configured on the docs side):
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/src/services/bootstrap/branding.constant.ts)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)

### CODE REVIEW

**Comment by dkwon17** [comment]
Here is a preview:
image

**Review by ibuziuk — APPROVED** [review]

**Comment by dkwon17** [comment]
I'm not sure if these are considered QE tests (for the checkbox), but the following PRs have added tests for the scenarios that the documentation describes:
https://github.com/eclipse-che/che-operator/pull/940
https://github.com/eclipse-che/che-operator/pull/880


image

**Comment by MichalMaler** [comment]
Will check this today :)

**Comment by MichalMaler** [comment]
My review is here:
https://github.com/dkwon17/che-docs/pull/2/

**Review by MichalMaler — APPROVED** [review]

**Comment by MichalMaler** [comment]
LGTM.
Just please, check those steps one more time:

1) Edit the `CheCluster` Custom Resource and set `.spec.imagePuller.enable` to `true`:
2) Uninstall {image-puller-name-short} using {prod-short} Operator:
3) Configure {image-puller-name-short} in the `CheCluster` Custom Resource:

If that is OK, just please delete that accidentally added file and we are ready to go :).

**Comment by MichalMaler** [comment]
Patch applied

**Comment by dkwon17** [comment]
The new changes look great, I have also removed the unnecessary file

File: modules/administration-guide/partials/proc_installing-image-puller-using-che-operator.adoc [hunk 1]
```diff
@@ -15,9 +15,10 @@ include::example$snip_{project-context}-installing-image-puller-using-che-operat
 
 .Procedure
 
-. Edit the `CheCluster` Custom Resource and set `.spec.imagePuller.enable` to `true`
+. Edit the `CheCluster` Custom Resource and set `.spec.imagePuller.enable` to `true`:
++
+.. Enable {image-puller-name-short} in the `CheCluster` Custom Resource:
```
Comment (dkwon17) [inline modules/administration-guide/partials/proc_installing-image-puller-using-che-operator.adoc]: Sorry, spoke to soon, could we combine these two lines (line 18 and 20) to something like:
```
. Enable {image-puller-name-short} in the `CheCluster` Custom Resource by setting `.spec.imagePuller.enable` to `true`:
```

Comment (MichalMaler) [reply modules/administration-guide/partials/proc_installing-image-puller-using-che-operator.adoc]: Yeah, I was thinking about it too.

**Comment by ibuziuk** [comment]
@MichalMaler @themr0c build check is failing on the timeout (Java Lombok  website is down). How shall we proceed?

**Comment by MichalMaler** [comment]
@ibuziuk Hello Ilya. I am aware of that, but dont know how to overpass this yet. Fabrice is on PTO.

**Comment by ibuziuk** [comment]
@MichalMaler can we exclude the check for link validation or point to the right one -  https://docs.openshift.com/container-platform/latest/builds/setting-up-trusted-ca.html ?

**Comment by MichalMaler** [comment]
> @MichalMaler can we exclude the check for link validation or point to the right one - https://docs.openshift.com/container-platform/latest/builds/setting-up-trusted-ca.html ?

Will try to do the combination of both. Good idea.

**Pull request merged**